Backpack-SwiftUI/Text/Classes/BPKText.swift

```diff
@@ -0,0 +1,87 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2022 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+fileprivate extension Text {
+    func font(style: BPKFontStyle) -> some View {
+        let font = style.font
+        return self.font(Font(font))
+            .tracking(style.letterSpacing)
+            .lineSpacing(style.lineHeight - font.lineHeight)
+    }
+}
+
+/// A view that displays one or more lines of text
+/// By default the color of BPKText is set to `BPKColor.textPrimary` with a lineLimit of 1
+///
+/// Use `foregroundColor(_ color: BPKColor)` to change the color of the text
+/// Use `limeLimit(_ number: Int?)` to set the number of lines can be displayed. Pass in `nil` for unlimited lines
+///
+public struct BPKText: View {
+    private var text: String
+    private var style: BPKFontStyle
+    
+    private var textColor = Color(BPKColor.textPrimaryColor)
+    private var lineLimit: Int? = 1
```

Comment:
would a default value of nil be more representative of the native API?

Reply:
The original BPKLabel has a limit of 1, that's why I have added it to our BPKText.

Reply:
https://developer.apple.com/documentation/uikit/uilabel/1620539-numberoflines